src/tasks/tasks.service.ts

```diff
@@ -22,4 +22,8 @@ export class TasksService {
   async findAllTasks(): Promise<Task[]> {
     return await this.tasksRepository.findAll();
   }
+
+  async findTaskById(id: number): Promise<Task | null> {
```

Comment:
```suggestion
  async findOne(id: number): Promise<Task | null> {
```

Can name this findOne. It's not necessary to explicitly put tasks in the service method names since you'd be calling this method in other files with tasksService like so: `this.tasksService.findOne`